lib/CtTask.type.d.ts

```diff
@@ -0,0 +1,43 @@
+export interface Task {
```

Comment:
If you use declare
```suggestion
declare interface Task {
```
Then in the [lib/AppCMSClient.d.ts](lib/AppCMSClient.d.ts) you won't have yo use an import.